modules/running-network-verification-manually-cli.adoc

```diff
@@ -19,18 +19,42 @@ When you run the network verification, you can specify a set of VPC subnet IDs o
 
 .Procedure
 
-* Verify the network configuration by using one of the following methods: 
+* Verify the network configuration by using one of the following methods:
 ** Verify the network configuration by specifying the cluster name. The subnet IDs are automatically detected:
 +
 [source,terminal]
 ----
-$ rosa verify network -c <cluster_name> <1>
+$ rosa verify network --cluster <cluster_name> <1>
 ----
 <1> Replace `<cluster_name>` with the name of your cluster.
 +
 .Example output
 [source,terminal]
 ----
+I: Verifying the following subnet IDs are configured correctly: [subnet-0b761d44d3d9a4663 subnet-0f87f640e56934cbc]
+I: subnet-0b761d44d3d9a4663: pending
+I: subnet-0f87f640e56934cbc: passed
+----
+*** Ensure that verification to all subnets has been completed.
++
+[source,terminal]
+----
+$ rosa verify network --watch <1>
+                      --status-only <2>
+                      --region us-east-1 <3>
+                      --subnet-ids subnet-0b761d44d3d9a4663,subnet-0f87f640e56934cbc <4>
+----
+<1> The `watch` flag causes the command to complete after all the subnets under test are in a failed or passed state.
+<2> The `status-only` flag does not trigger a run of network verification but returns the current state that is pending, for example, `subnet-123 (verification still in-progress)`. By default, without this option, a call to this command always triggers a verification of the specified subnets.
```

Comment:
This description is not correct. It will return the current status. Not necessarily "pending"
> but returns the current state that is pending

Reply:
Thanks I will change this.